index.js

```diff
@@ -82,6 +82,9 @@ exports.register = function (server, options, next) {
                 }, function(err, result){
 
                     if(err){
+                        if (debug) {
+                            internals.log('error', err.formatted);
```

Comment:
Actually now that I look at this more, `err.formatted` seems to be an `undefined` property. Not sure what's intended there. 

[According to the docs](https://github.com/sass/node-sass#error-object) on node-sass, it has the following props:

- message (String) - The error message.
- line (Number) - The line number of error.
- column (Number) - The column number of error.
- status (Number) - The status code.
- file (String) - The filename of error. In case file option was not set (in favour of data), this will reflect the value stdin.

Perhaps we want some combination of message, line:column, file?